linkatos/message.py

```diff
@@ -13,3 +15,16 @@ def contains_a_link(message):
         answer = answer.group(0).strip()
 
     return answer
+
+
+yes_re = re.compile("(Yes|YES|yes)")
```

Comment:
I prefer having variable bindings atop the functions. Sort of a file declaration.